src/ui/test/CMakeLists.txt

```diff
@@ -44,9 +44,11 @@ if(ENABLE_IMGUI_TEST_ENGINE)
               imgui-node-editor
               imgui_tests_common)
     add_test(NAME ${NAME} COMMAND ${NAME})
+    target_include_directories(${NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/..
```

Comment:
IMO the better way here would be to add the `raii_wrapper` target to `target_link_libraries`. Probably better to clean this up treewide separately though.